zmon_cli/main.py

```diff
@@ -133,47 +135,82 @@ def check_workers(r, workers):
             error(e)
 
 
+def query_password(user):
+    pw = click.prompt("Password for {}".format(user), hide_input=True)
+    keyring.set_password("zmon-cli", user, pw)
+    return pw
+
+
 def get_config_data():
     fn = os.path.expanduser(DEFAULT_CONFIG_FILE)
     data = {}
-    if os.path.exists(fn):
-        with open(fn) as fd:
-            data = yaml.safe_load(fd)
-    else:
-        raise Exception("Config file not found: ~/.zmon-cli.yaml")
+    try:
+        if os.path.exists(fn):
+            with open(fn) as fd:
+                data = yaml.safe_load(fd)
+            if 'password' in data:
+                keyring.set_password("zmon-cli", data['user'], data['password'])
+                del data['password']
+                with open(fn, mode='w') as fd:
+                    yaml.safe_dump(data, fd)
+        else:
+            click.secho("No configuration file found at [%s]".format(DEFAULT_CONFIG_FILE), fg="yellow")
```

Comment:
Please use the warning function from clickclick https://github.com/zalando/python-clickclick/blob/master/clickclick/console.py#L60